library/src/app/components/bank-account-management/bank-account-connect/bank-account-connect.component.ts

```diff
@@ -228,17 +228,6 @@ export class BankAccountConnectComponent implements OnInit {
         : metadata.accounts[0].iso_currency_code;
     const account = metadata.accounts[0];
 
-    console.log(this.config.environment);
-    console.log(
-      this.config.environment == 'demo'
-        ? 'USD'
-        : metadata.accounts[0].iso_currency_code
-    );
-    console.log(asset);
-    console.log(metadata.accounts);
-    console.log(isValidAsset(asset));
-    console.log(isOnlyAccount(metadata.accounts));
```

Comment:
After this merges please go test manually to ensure it's still working and that these logs didn't have some unintended side effect that made it work.